lte/gateway/c/oai/tasks/mme_app/mme_app_main.c

```diff
@@ -466,6 +467,8 @@ static int handle_message(zloop_t* loop, zsock_t* reader, void* arg) {
   put_mme_nas_state();
   put_mme_ue_state(mme_app_desc_p, imsi64);
 
+  OAILOG_INFO(LOG_MME_APP, "zsys_interrupted: %d", zsys_interrupted);
```

Comment:
we also want this down under zloop_start, before the AssertFatal("Asserting as mme_app_thread should not be exiting on its own")